src/DatePicker/component/DatePickerTypes.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+import * as React from 'react';
+
+export type Props = {|
+  +isVisible: boolean, // this prop is supported only on mobile
+  +mode?: 'date' | 'time' | 'datetime',
+  +datePickerMode?: 'calendar' | 'spinner' | 'default', // this prop is supported only on android
+  +date?: Date,
+  +minDate?: Date,
+  +maxDate?: Date,
+  +onConfirm: Date => void,
+  +onDismiss: () => void,
+  +customInput?: React.Node, // this prop is supported only on web
```

Comment:
I am just wondering here if it would make sense to do the following:

```
type Props = {|
  +mobile?: {|
       +isVisible: boolean, // this prop is supported only on mobile
  |},
  +android?: {|
     +datePickerMode?: 'calendar' | 'spinner' | 'default', // this prop is supported only on android
   |},
  +web?: {|
     +customInput?: React.Node, // this prop is supported only on web
   |},
  +mode?: 'date' | 'time' | 'datetime',
  +date?: Date,
  +minDate?: Date,
  +maxDate?: Date,
  +onConfirm: Date => void,
  +onDismiss: () => void,
|};
```

so that it's clearer to the consumer which props are accessible on which platform?
@tbergq Thoughts?

Reply:
No, I don't think we should do `mobile` `web` `android` it is ok to have some optional props just valid for one platform. Even react-native has that. 

It should be documented somewhere somehow though.

Reply:
So what we need is some documents page like [orbit](https://orbit.kiwi/)